pkg/file/span.go

```diff
@@ -0,0 +1,42 @@
+package file
+
+import (
+	"math"
+
+	"github.com/ethersphere/bee/pkg/swarm"
+)
+
+var Spans []int64
+
+func init() {
+	Spans = GenerateSpanSizes(9, swarm.Branches)
+}
+
+// generates a dictionary of maximum span lengths per level represented by one SectionSize() of data
+//
+// TODO: level 0 should be SectionSize() not Branches()
```

Comment:
is this TODO still relevant? also, the godoc doesn't start with the function name

Reply:
actually I'm not entirely sure :D this is copied from the other code I wrote, and I since forgot exactly what I meant. Since it works, I'll remove it.